src/04_printing.py

```diff
@@ -11,7 +11,10 @@
 # Using the printf operator (%), print the following feeding in the values of x,
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
+print("x is %s, y is %s, z is %s" % (x, y, z))
```

Comment:
there is a way to round the output of printed floats, search around and let me know if you don't figure it out.

Reply:
Fixed!